website/docs/r/search_analyzer.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: search analyzer"
+sidebar_current: "docs-mongodbatlas-resource-search-analyzer"
+description: |-
+Provides a Search Analyzers resource.
+---
+
+# mongodbatlas_search_analyzer
+
+`mongodbatlas_search_analyzer` creates new user-defined Atlas Search [analyzers](https://docs.atlas.mongodb.com/reference/atlas-search/analyzers/#std-label-analyzers-ref) and replaces existing ones. If you have any user-defined analyzers for your Atlas Search [index](https://docs.atlas.mongodb.com/atlas-search/#std-label-fts-top-ref), they will be replaced by the new analyzer or analyzers specified in this request.
```

Comment:
But how do you then link the analyzer in the index since these are first created normally in the index creation if I understand the API right?

Reply:
This resulted from a mis-understanding on my part. Ignore.